clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/cache/ApplicationCacheClientTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.cache;
+
+import static com.netflix.spinnaker.clouddriver.ecs.cache.Keys.Namespace.APPLICATIONS;
+import static org.junit.Assert.assertTrue;
```

Comment:
Can we use assertTrue from either **junit.framework.TestCase.assertTrue** or **org.junit.Assert.assertTrue**
It gives some unexpected behaviors if you mix and match them within a same package. It caused me problems previously, so just a suggestion.

Reply:
These are unit tests so we do expect to use org.junit.Assert.assertTrue

Reply:
I could see junit.framework.TestCase.assertTrue  [here](https://github.com/spinnaker/clouddriver/pull/5234/commits/934be3fbc07ad3f539ca01849b4fe8aa6f734074#diff-17c3e0e85f30e5215489b55e49ceda0aeb21648b5c0b73eb8e0cc097d9ff5186R21)

Reply:
Sry the link isn't taking me anywhere.

Reply:
I had given reference to last commit, so sorry. I have fixed the link w.r.t new commit.

Reply:
Those are 2 different packages